files/system/usr/etc/authselect/custom/secureblue/REQUIREMENTS

```diff
@@ -0,0 +1 @@
+- make sure fprintd service is configured and enabled
```

Comment:
what will happen if it isn't?

Reply:
fingerprint auth should simply not work but it shouldn't breake anything

Reply:
cool